.gitlab-ci.yml

```diff
@@ -564,7 +553,6 @@ test-suite:edge:dune:dev:
     paths:
       - _build/log
       - _build/default/test-suite/logs
-    expire_in: 2 week
```

Comment:
This one was shorter but I don't think 2 more weeks of logs will be what overflows storage